Source/EasyNetQ.Logging.Serilog/LoggerFactory.cs

```diff
@@ -0,0 +1,43 @@
+using S = Serilog;
+
+namespace EasyNetQ.Logging.Serilog
+{
+    /// <summary>
+    ///     Creates ILogger
+    /// </summary>
+    public interface ILoggerFactory
```

Comment:
duplicated interface?

Reply:
Who needs this interface? It is superfluous as similar in the other project.

Reply:
Fixed